resources/js/Pages/Admin/SchoolsPanel.vue

```diff
@@ -1,11 +1,126 @@
 <script setup lang="ts">
 import {Head} from '@inertiajs/vue3'
+import { PlusCircleIcon } from '@heroicons/vue/20/solid'
+import AddressInput from '@/components/Common/AddressInput.vue'
+import CrudPage from '@/components/Crud/CrudPage.vue'
+import Expand from '@/components/Common/Expand.vue'
+import InputWrapper from '@/components/QuizzesPanel/InputWrapper.vue'
+import {router} from '@inertiajs/vue3'
+import {ref, watch} from 'vue'
+import axios from 'axios'
+import ButtonFrame from '@/components/Common/ButtonFrame.vue'
+import vDynamicTextAreaHeight from '@/Helpers/vDynamicTextAreaHeight'
+import vDynamicInputWidth from '@/Helpers/vDynamicInputWidth'
+
+defineProps<{schools: School[]}>()
+
+const sortOptions: SortOptionConstructor[] = [
+  { text: 'Po nazwie (A–Z)', type: 'name' },
+  { text: 'Po nazwie (Z–A)', type: 'name', desc: true },
+  { text: 'Od najnowszych' , type: 'creationDate' },
+  { text: 'Od najstarszych', type: 'creationDate', desc: true },
+  { text: 'Od najnowszych zmienionych', type: 'modificationDate' },
+  { text: 'Od najstarszych zmienionych', type: 'modificationDate', desc: true },
+]
+
+const status = ref(false)
+
+setInterval(() => {
+  axios.get('/admin/schools/status')
+    .then(res => status.value = res.data.done)
+    .catch(() => status.value = false)
+}, 1000)
+
+watch(status, () => router.reload())
+
+function startFetching() {
+  status.value = false
+  axios.post('/admin/schools/fetch')
+}
 </script>
 
 <template>
   <Head>
-    <title>Szkoły</title>
+    <title>Szkoły - Panel administracyjny</title>
   </Head>
 
-  Quizzes - CRUD
+  <CrudPage
+    :options="sortOptions"
+    :items="schools"
+    resource-name="schools"
+    new-button-text="Dodaj szkołe"
+    :new-item-data="{ name: 'Nowa Szkoła', regon: '', apartmentNumber: '', street: '', buildingNumber: '', city: '', numberOfStudents: 0, zipCode: '' }"
+    deletable
+    mobile-nav
+  >
+    <template #actions>
+      <Expand />
+
+      <ButtonFrame
```

Comment:
Since you are using `ButtonFrame` as a button, maybe need to get rid of the `Button` component?

Reply:
I changed ButtonFrame to Button. Button sounds more intuitive and immediately makes it clear that this component is a button.